texts/issues/messages.yml

```diff
@@ -1,63 +1,444 @@
+    unable to find any full text links
+  resolution: |
+    Ensure that dc:identifier contains a direct link to the full text. We only
+    support the following file types: .pdf
+  details: [currentUrl]
+
 -
-  type: EXTERNAL_UNKNOWN
-  title: No full text was found due to unknown external reasons
+  hidden: true
+  type: NO_ATTACHMENT_FOUND_NO_REPORTED_FAILURES
+  matches: [NON_EXISTENT_PAGE_ATTACHMENT]
+  severity: WARNING
+  trigger: |
+    This issue is triggered when no specific issue was detected and reported.
+
+    If this issue is stored, it means we need to identify and find the root
+    cause.
+
+-
+  hidden: true
+  type: ATTACHMENT_PROCESSING_FAILURE
+  matches: []
+  severity: ERROR
+
 -
-  type: ENCRYPTED_ATTACHMENT
-  title: Full text has digital restrictions
+  type: ATTACHMENT_ENCRYPTED
+  matches: [ENCRYPTED_ATTACHMENT]
+  severity: ERROR or WARNING
+  trigger: |
+    This issue is triggered when a pdf has DRM enabled.
+
+    If we are able to bypass, a Warning is given
+
+    If we are not, an Error
+  title: The pdf is encrypted
+  description: |
+    The pdf has Document Security enabled. This digitally restricts how the
+    pdf is opened and used. For example, it may prevent editing, printing or
+    viewing the full text via a screen reader.
+
+    If this is message is issued with a Warning, we were able to bypass the
+    security settings
+
+    If this message is an error, we were unable to bypass the security and we
+    have not imported the fulltext.
+  resolution: |
+    We recommend that Document Security is removed from the PDF.
+    [https://helpx.adobe.com/uk/acrobat/using/securing-pdfs-passwords.html#remove_password_security]
+  details: [currentUrl]
+
 -
   type: ATTACHMENT_TITLE_MISMATCH
-  title: Full text title mismatches metadata
+  severity: WARNING
+  trigger: |
+    We check that the pdf contains the title provided in the metadata. If the
+    first 50 lines of the pdf do not contain an exact match of the title OR is
+    similar.
+  title: Title match issue
+  description: |
+    To ensure we link the correct metadata and full text, we check that the
+    metadata title text is contained within the pdf. An exact match is not
+    required, but must be similar.
+
+    This error is common or scanned items where Optical Character Recognition
+    (OCR) has not been performed.
+  resolution: |
+    Ensure that the Metadata Title is the same as the document title.
+
+    For example:
+
+    Failed match example:
+
+    dc:title:
+    An Open Access Paper: a study
+
+    However, the paper may have the title:
+
+    A study of Open Access Papers
+
+    A successful partial match:
+
+    dc:title
+    A study of Open Access Papers in Europe
+
+    Studying Open Access papers in Europe
+
+    The closer the match, the more likely we will accept the change.
+
+
+    You may bypass this check by including a link to the full text in
+    dc:identifier
```

Comment:
```suggestion
  title: Full text title mismatches metadata
  description: |
    To ensure we link the correct metadata and full text, we check that the
    metadata title text is contained within the PDF document.
    An exact match is not required, but must be similar.

    This error is common for scanned items where Optical Character
    Recognition&nbsp;(OCR) has not been performed.

    Ensure that the Metadata Title is the same as the document title
    or including a link to the full text in `dc:identifier`.
  resolution:
    -
      caption: Mark resolved
  # should be moved to separate place or deleted
  extra-intructions: |
    Ensure that the Metadata Title is the same as the document title. For
    example:

    Failed match example:
    - `dc:title:` _An Open Access Paper: a study_
    - PDF title: _A study of Open Access Papers_

    A successful partial match:
    - `dc:title`: _A study of Open Access Papers in Europe_
    - PDF title: _Studying Open Access papers in Europe_

    The closer the match, the more likely we will accept the change.

    You may bypass this check by including a link to the full text
    in `dc:identifier`.
```

Reply:
Line 269: PDF documents to full text
Line 274: is `resolution` missing from this line?

Reply:
Line 269: 👍 changed

Line 274: No, it's not. The idea is to have a single description and under the resolution have a set of action buttons that either go to some kind of settings, instructions or dismiss the issue.